### PR TITLE
Add steps for beta assignment 3

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+### Download the sample application
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+Clone or download the Android Sample application from [https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3).
+
+To get started, follow the steps described in [AEP SDK Sample App Android - Installation](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-1#installation).
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md:24]: There is no `beta-assignment-1` branch for this repo, but there is a tag for it. Did you intend to reference the tag instead?

File: beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### Swift
+
+Clone or download the iOS Swift Sample application from [https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-3](https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-3).
+
+To get started, follow the steps described in [AEP SDK Sample App Swift - Installation](https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-1#installation).
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md:33]: Same, there is no `beta-assignment-1` branch, but there is a tag. Should this reference the tag or do you intend to create a branch?

File: beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+* Completion of [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup).
+
+### Download the sample application
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+Clone or download the Android Sample application from [https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3).
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md:22]: Add code sample on how to clone the specific tag/branch:
`git clone git@github.com:adobe/aepsdk-sample-app-android.git --branch beta-assignment-1 --single-branch`

For downloading the zip, If we are using tags, we should link to the tag and point out the download link.
https://github.com/adobe/aepsdk-sample-app-android/releases/tag/beta-assignment-1

File: beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+
+To get started, follow the steps described in [AEP SDK Sample App Android - Installation](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-1#installation).
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### Swift
+
+Clone or download the iOS Swift Sample application from [https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-3](https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-3).
```
Comment (kevinlind) [inline beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md:31]: Same comment as about about adding specific instructions on how to clone a specific tag/branch and include the direct link to download the tag .zip.

File: beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md (hunk 1 above)
Comment (emdobrin) [reply beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md]: I added the download link instead, let me know what do you think.

File: beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+   * In the Schema structure, select the  `Product review` mixin and click on the top level `+`.
+
+   * Start adding fields as follows:
+
+     | Field name | Display name | Type    | Required |
+     | :--------- | :----------- | :------ | -------- |
+     | productSku | Product SKU  | String  | Yes      |
+     | reviewText | Review Text  | String  | No       |
+     | rating     | Rating       | Integer | Yes      |
+     | reviewerId | Reviewer ID  | String  | No       |
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md]: I marked it as required for now, we will clarify this in the next days.

**Review by kevinlind — APPROVED** [review]

**Pull request merged**